src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2581,8 +2581,6 @@ namespace NServiceBus.Routing.MessageDrivenSubscriptions
     {
         public Publishers() { }
         public void Add(System.Type eventType, string publisher) { }
-        public void Add(System.Reflection.Assembly eventAssembly, string publisher) { }
```

Comment:
So we explicitely don't want these APIs? 

I assume the reasoning is that what happens if we get passed in an assembly which is not scanned by the assembly scanner, is that correct?

Because it theory we could still support those APIs by essentially union in that assembly in the scanning mechanism.